### ISSUE TITLE
windows install mess?

### ISSUE THREAD

**Issue by berl** [napari/napari#75]
## 🐛 Bug

The symlink in the repo doesn't work for windows. in my conda environment, pip install exits without error but does not result in an installed package available in python (also demos don't run).  `python setup.py install` fails with the error below.

## To Reproduce

1. clone repo, cd into napari-gui
2. python setup.py install

```
python setup.py install
running install
running bdist_egg
running egg_info
creating napari_gui.egg-info
writing napari_gui.egg-info\PKG-INFO
writing dependency_links to napari_gui.egg-info\dependency_links.txt
writing requirements to napari_gui.egg-info\requires.txt
writing top-level names to napari_gui.egg-info\top_level.txt
writing manifest file 'napari_gui.egg-info\SOURCES.txt'
reading manifest file 'napari_gui.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'napari_gui\_version.py'
warning: no files found matching 'napari_gui\elements\qt\icons'
writing manifest file 'napari_gui.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build
creating build\lib
creating build\lib\examples
copying examples\demo.py -> build\lib\examples
copying examples\slider.py -> build\lib\examples
copying examples\__init__.py -> build\lib\examples
UPDATING build\lib\napari_gui/_version.py
error: [WinError 3] The system cannot find the path specified: 'build\\lib\\napari_gui/_version.py'
```
## Expected behavior
expected behavior (i.e.  demo.py runs) can be obtained by creating a Windows shortcut and then manually copying the contents from gui to the build\\lib\\napari_gui directory.  after this, running `python setup.py install` works and the demo runs.

## Environment

 - napari-gui==0.0.4+45.g6e15cf8
 - OS (e.g., Linux): Windows 10
 - Python version: 3.6.1

**Comment by sofroniewn** [comment]
hi brian - sorry about that. i havn't got a windows machine, but I'm sure someone else will. thanks for flagging, we'll get back to you! it's strange to me that it can't find those files as they do exist!

**Comment by berl** [comment]
yeah, they're just not getting copied, probably because the symlink/shortcut isn't doing what it's supposed to. It seems like an odd choice to not just rename the directory as `napari-gui` and do away with the symlink. but it looks like someone tried that at some point and changed it back?

**Comment by sofroniewn** [comment]
ah actually i think i was told by @kne42 to do something with putting the icons folder in our requirements somewhere to make sure it got copied with pip - but I didn't do it and can't quite remember now what it was i was supposed to do. might not be exactly the problem here, but probably worth doing regardless

**Comment by kne42** [comment]
@berl Thank you for the bug report! As you surmised, it does appear to be an issue with using Unix-style symlinks on Windows (see [this thread](https://stackoverflow.com/questions/5917249/git-symlinks-in-windows) on StackOverflow).

IMO the easiest solution is to just implement the structure as proposed in Napari/napari-core#26 if everyone agrees on that.

In the meantime, creating a [Windows-style symlink](https://docs.microsoft.com/en-us/windows/desktop/fileio/creating-symbolic-links) from `gui` to `napari_gui` should do the trick.

I ended up choosing to use a symlink because I wanted the package to be both `pip`-installable and a plugin (before we agreed on it being a standalone entity) while trying not to use the common `gui` namespace. There was an option in `setup.py` to do this renaming automatically but unfortunately a bug when running it in development mode excluded this from being a viable option.

@sofroniewn No that was `MANIFEST.in` and you did put it in there :P

**Comment by sofroniewn** [comment]
@kne42 thanks!! I had no recollection of that - but I now see it's there :-)

**Comment by kne42** [comment]
fixed in #151
